src/pages/junghoon/Login/Login.js

```diff
@@ -1,13 +1,108 @@
-import React, { Component } from 'react'
+import React, { Component } from "react";
+import "./Login.scss";
+import "../../../styles/common.scss";
 
 class Login extends Component {
+  constructor(e) {
+    super(e);
+
+    this.state = {
+      userId: "",
+      userPw: "",
+    };
+  }
+
+  // goToMain = () => {
+  //   this.props.history.push("/mainPark");
+  // };
+
+  handleIdInput = (e) => {
+    this.setState({
+      userId: e.target.value,
+    });
+  };
+
+  handlePwInput = (e) => {
+    this.setState({
+      userPw: e.target.value,
+    });
+  };
+
+  onClick = () => {
+    fetch("http://10.58.6.173:8000/user/signin", {
+      method: "POST",
+      body: JSON.stringify({
+        email: this.state.userId,
+        password: this.state.userPw,
+        name: "정훈1121",
+        phone: "01048780454",
+        nickname: "나1나나112",
+      }),
+    })
+      .then((response) => response.json())
+      .then((result) => {
+        console.log(result);
+        if (result.message === "SUCCESS") {
+          this.props.history.push("/mainPark");
+          localStorage.setItem("TOKEN", result.token);
+        } else {
+          alert("리트라이 플리즈");
+        }
+      });
+  };
+
+  checkSubmmit = () => {
+    const { userId, userPw } = this.state;
+    return userId.includes("@") > 0 && userPw.length >= 5;
+  };
+
   render() {
+    // const isEnabled = this.checkSubmmit();
+    // console.log(isEnabled);
+
     return (
-      <div>
-        Login
+      <div className="LogInPageAll">
+        <main className="mainLogIn">
+          <header className="mainLogo">
+            <p>Westagram</p>
+          </header>
```

Comment:
```suggestion
            <h1>Westagram</h1>
```
1. header 태그는 알맞지 않아 보이네요!
2. header 안에 p 태그를 쓰기 보단 하나의 태그만 사용해주세요! 불필요한 depth 입니다.